nextflow.config

```diff
@@ -111,15 +110,13 @@ try {
 profiles {
     debug { process.beforeScript = 'echo $HOSTNAME' }
     conda {
-        conda.enabled          = true
```

Comment:
why are you removing this?

Reply:
possibly deprecated? it looks like the `conda` configuration scope no longer includes an option `enabled`

https://www.nextflow.io/docs/latest/config.html#scope-conda

Reply:
I thought this was just fix in December with adding this https://github.com/nf-core/tools/blob/dev/nf_core/pipeline-template/nextflow.config (after the `enabled_conda` param wasn't necessary anymore) 🤔 . If that is the case, should be added to the template too, I guess

Reply:
Yeah sorry, I think I confused it with that enabled_conda param that was deprecated...